### DISCUSSION TITLE
Welcome to osint-tools Discussions!

### DISCUSSION THREAD

**Discussion by Astrosp** [Astrosp/osint-tools#3] [Ideas]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.
  * You can add missing tools  

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by uhmhc** [comment]
hey bro! how can I contact you, please? I have a great offer for you, as I'm looking for someone to build me an Osint system

**Reply by Astrosp** [reply]
Mail me in astrosp@proton.me

**Comment by hariharan136** [comment]
Hi bro how to use this osint code plz tell me

**Reply by Astrosp** [reply]
This is just a list of tools, there is sections for each type click on what you want and you get links for that type.
